README.md

```diff
@@ -28,39 +28,40 @@ The filenames contain the intended transcription.
 
 As a quick start, these commands are provided as examples of how to use the binary.  They should be run from the current directory (`cli-client/`).
 
-Note: These commands assume that the cubic server instance is available at `localhost:2727`.  For initial testing, there is a demo server available at `http://demo-cubic.cobaltspeech.com:2727`.
+Note: These commands assume that the cubic server instance is available at `localhost:2727`.
+
+For quick testing, Cobalt's demo server can be accessed at
+`http://demo-cubic.cobaltspeech.com:2727`.  This uses TLS and does not need the
+`--insecure` flag.  This server should only be used for small tests.  Access to
+the server maybe limited or restricted at times.  Data uploaded to this server
+may be stored for internal purposes.  Commercial use of the demo service is not
+allowed.
```

Comment:
I suggest breaking this into two paragraphs:

```suggestion
For quick testing, Cobalt's demo server can be accessed at http://demo-cubic.cobaltspeech.com:2727. This uses TLS so it does not need the --insecure flag. 

Commercial use of the demo service is not permitted. This server is for testing and demonstration purposes only and is not guaranteed to support high availability or high volume. Data uploaded to the server may be stored for internal purposes.
```

Reply:
added. also moved to the bottom of the file.